TestList/UnitTest1.cs

```diff
@@ -0,0 +1,60 @@
+// --------------------------------------------------------------------------------------------
+// Training ~ A training program for interns at Metamation, Batch - July 2023
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------------
+// Program.cs
+// MYLIST<T> 
+// Implement a custom MyList<T> class using arrays as the underlying data structure.
+// --------------------------------------------------------------------------------------------
+using Training;
+namespace LIST {
+   [TestClass]
+   public class UnitTest1 {
+      MyList<int> list = new ();
+      [TestMethod]
+      public void TestAdd () {
+         for (int i = 1; i <= 5; i++) list.Add (i);
+         Assert.AreEqual (5, list.Count);
+         Assert.AreEqual (8, list.Capacity);
+         list[1] = 4;

```

Comment:
Write test cases to compare `MyList<T>` with the actual `List<T>`. Assert if both are equal. Change all test cases this way.